api/app/settings/local.py

```diff
@@ -9,3 +9,9 @@
 MIDDLEWARE.extend(["debug_toolbar.middleware.DebugToolbarMiddleware"])
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+# overwrite swagger settings to allow access without authentication
+SWAGGER_SETTINGS = {
```

Comment:
Rather than redefining the `SHOW_REQUEST_HEADERS` key, couldn't we achieve this with the following single line instead? 

```python
SWAGGER_SETTINGS["USE_SESSION_AUTH"] = False
```